precode.sh

```diff
@@ -1,43 +1,75 @@
 #!/bin/bash
-
 # создаём каталог task с вложенными директориями
 # task
 #   dir1
 #   dir2
 #   dir3
-#       dir4
+#   dir4
+
+mkdir task && mkdir task/dir{1..3} && mkdir -p task/dir3/dir4
```

Comment:
Используя изначально флаг `-p` можно так сократить
```suggestion
mkdir -p task/dir{1..3} task/dir3/dir4
```